examples/src/main/java/tachyon/examples/BasicRawTableOperations.java

```diff
@@ -49,10 +50,18 @@
   private long mId;
 
   public BasicRawTableOperations(TachyonURI masterAddress, TachyonURI tablePath,
-      TachyonStorageType writeType) {
+      TachyonStorageType tachyonWriteType, UnderStorageType ufsWriteType) {
     mMasterAddress = masterAddress;
     mTablePath = tablePath;
-    mWriteType = writeType.isStore() ? WriteType.MUST_CACHE : WriteType.THROUGH;
+    if (tachyonWriteType.isStore()) {
+      if (ufsWriteType.isSyncPersist()) {
```

Comment:
`mWriteType = ufsWriteType.isSyncPersist() ? WriteType.CACHE_THROUGH : WriteType.MUST_CACHE` may be cleaner?